Cargo.toml

```
@@ -1,32 +1,26 @@
 [workspace]
 resolver = "2"
 
-members = [
-    "drink",
-    "drink/test-macro",
-    "drink-cli",
-]
+members = ["drink", "drink/test-macro", "drink-cli"]
 
-exclude = [
-    "examples/",
-]
+exclude = ["examples/"]
 
 [workspace.package]
-authors = ["Cardinal"]
+authors = ["Use Ink <ink@use.ink>"]
```

```suggestion
authors = ["Cardinal", "Use Ink <ink@use.ink>"]
```
Ditto about authors.